### DISCUSSION TITLE
error=ssl_required

### DISCUSSION THREAD

**Discussion by t-sharabi** [apiman/apiman#2418] [Help]
Hello

after doing the quick setup on AZURE vm for testing , I have configured the .env file (APIMAN_HOSTNAME=domain.com) and i have the following results:

1. http://apiman.domain.com:8080/apimanui === we are sorry HTTPS required
2. https://apiman.domain.com:8080/apimanui === 404 page not found

so kindly i need your help

**Comment by msavy** [comment]
Happy New Year!

Sorry, I don't have access to Azure at present, and the information you have provided is not sufficient to give a definitive answer.

Google suggests that the error might be to do with Keycloak configuration.

**Reply by msavy** [reply]
@t-sharabi did you resolve this? Can you provide any context that might help other users on Azure?

**Reply by t-sharabi** [reply]
hello
sorry for the late reply, unfortunately tell now I couldn't solve it, the issue is that I am not able install the keyclock docker properly per to the instructions in the documentation, I will get back to work on it this week and will update the topic accordingly 

thanks
